tests/transports-governance/conftest.py

```
@@ -0,0 +1,476 @@
+    "teams": [],
+    "virtual_keys": [],
+}
+
+
+def track_created_entity(entity_type: str, entity_id: str):
+    """Track created entity for cleanup"""
+    if entity_type in created_entities:
+        created_entities[entity_type].append(entity_id)
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Ruff (0.12.2)

`scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)